migrations/drivers.go

```diff
@@ -16,95 +16,238 @@ import (
 	}
-	return "NOW()"
+
+	return "" // placeholder
 }
 
 func (db *datastore) typeInt() string {
-	if db.driverName == driverSQLite {
+	switch db.driverName {
+	case driverSQLite:
 		return "INTEGER"
+	case driverMySQL:
+		return "INT"
+	case driverPostgres:
+		return "INT"
 	}
-	return "INT"
+
+	return "" // placeholder
 }
 
 func (db *datastore) typeSmallInt() string {
-	if db.driverName == driverSQLite {
+	switch db.driverName {
+	case driverSQLite:
 		return "INTEGER"
+	case driverMySQL:
+		return "SMALLINT"
+	case driverPostgres:
+		return "SMALLINT"
 	}
-	return "SMALLINT"
+
+	return "" // placeholder
 }
 
 func (db *datastore) typeTinyInt() string {
-	if db.driverName == driverSQLite {
+	switch db.driverName {
+	case driverSQLite:
 		return "INTEGER"
+	case driverMySQL:
+		return "TINYINT"
+	case driverPostgres:
+		return "SMALLINT"
 	}
-	return "TINYINT"
+
+	return "" // placeholder
 }
 
 func (db *datastore) typeText() string {
 	return "TEXT"
 }
 
 func (db *datastore) typeChar(l int) string {
-	if db.driverName == driverSQLite {
+	switch db.driverName {
+	case driverSQLite:
 		return "TEXT"
+	case driverMySQL:
+		return fmt.Sprintf("CHAR(%d)", l)
+	case driverPostgres:
+		return fmt.Sprintf("CHAR(%d)", l)
 	}
-	return fmt.Sprintf("CHAR(%d)", l)
+
+	return "" // placeholder
 }
 
 func (db *datastore) typeVarChar(l int) string {
-	if db.driverName == driverSQLite {
+	switch db.driverName {
+	case driverSQLite:
 		return "TEXT"
+	case driverMySQL:
+		return fmt.Sprintf("VARCHAR(%d)", l)
+	case driverPostgres:
+		return fmt.Sprintf("VARCHAR(%d)", l)
 	}
-	return fmt.Sprintf("VARCHAR(%d)", l)
+
+	return "" // placeholder
 }
 
 func (db *datastore) typeVarBinary(l int) string {
-	if db.driverName == driverSQLite {
+	switch db.driverName {
+	case driverSQLite:
 		return "BLOB"
+	case driverMySQL:
+		return fmt.Sprintf("VARBINARY(%d)", l)
+	case driverPostgres:
+		return "BYTEA"
 	}
-	return fmt.Sprintf("VARBINARY(%d)", l)
+
+	return "" // placeholder
 }
 
 func (db *datastore) typeBool() string {
-	if db.driverName == driverSQLite {
+	switch db.driverName {
+	case driverSQLite:
 		return "INTEGER"
+	case driverMySQL:
+		return "TINYINT(1)"
+	case driverPostgres:
+		return "BOOLEAN"
 	}
-	return "TINYINT(1)"
+
+	return "" // placeholder
 }
 
 func (db *datastore) typeDateTime() string {
-	return "DATETIME"
+	switch db.driverName {
+	case driverSQLite:
+		return "DATETIME"
+	case driverMySQL:
+		return "DATETIME"
+	case driverPostgres:
+		return "TIMESTAMP"
+	}
+
+	return "" // placeholder
 }
 
 func (db *datastore) typeIntPrimaryKey() string {
-	if db.driverName == driverSQLite {
+	switch db.driverName {
+	case driverSQLite:
 		// From docs: "In SQLite, a column with type INTEGER PRIMARY KEY is an alias for the ROWID (except in WITHOUT
 		// ROWID tables) which is always a 64-bit signed integer."
 		return "INTEGER PRIMARY KEY"
+	case driverMySQL:
+		return "INT AUTO_INCREMENT PRIMARY KEY"
+	case driverPostgres:
+		return "SERIAL PRIMARY KEY"
```

Comment:
The more modern way to do this is

```suggestion
		return "INTEGER GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY"
```

cf. https://www.depesz.com/2017/04/10/waiting-for-postgresql-10-identity-columns/